src/components/consultation-pad-contents/consultation-pad-contents.tsx

```diff
@@ -162,7 +162,10 @@ export function ConsultationPadContents({
         {isRecording ? renderStopMic() : renderStartMic()}
         <Button
           className={styles.saveButton}
-          disabled={consultationText == '' && recordedText == ''}
+          disabled={
+            (consultationText == '' && recordedText == '') ||
+            isRecording == true
```

Comment:
would be an easy read to abstract it into its own function

Reply:
sure @N0-man  will update that